gc/base/GCExtensionsBase.hpp

```diff
@@ -1509,12 +1517,12 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 		, largePageWarnOnError(false)
 		, largePageFailOnError(false)
 		, largePageFailedToSatisfy(false)
-#if defined(OMR_GC_DOUBLE_MAP_ARRAYLETS)
 		, isArrayletDoubleMapRequested(false)
 		, isArrayletDoubleMapAvailable(false)
-#endif /* defined(OMR_GC_DOUBLE_MAP_ARRAYLETS) */
```

Comment:
These two fields are defined under the same flag. If `if defined` is removed here, it should be removed at fields definitions above as well.

Reply:
good observation. a preference is to keep the ifdefs if possible (if there are no spots in any current or more importantly new code that uses the variables without being wrapped with the same build flag)